### PR TITLE
openresolv: add gnused and dbus to PATH

### PR DESCRIPTION
**Pull request by amnore**
###### Description of changes

Openresolv uses sed ([here](https://github.com/NetworkConfiguration/openresolv/blob/master/resolvconf.in#L779)) and dbus ([here](https://github.com/NetworkConfiguration/openresolv/blob/master/dnsmasq.in#L206)), but they were not added to PATH. I added them in this PR.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/openresolv/default.nix [hunk 1]
```diff
@@ -27,7 +27,8 @@ stdenv.mkDerivation rec {
   installFlags = [ "SYSCONFDIR=$(out)/etc" ];
 
   postInstall = ''
-    wrapProgram "$out/sbin/resolvconf" --set PATH "${coreutils}/bin"
+    wrapProgram "$out/sbin/resolvconf" \
+      --set PATH "${coreutils}/bin:${gnused}/bin:${dbus}/bin"
```
Comment (nagy) [inline pkgs/tools/networking/openresolv/default.nix:31]: Can you use `lib.makeBinPath` ?

**Review by nagy — CHANGES_REQUESTED** [review]

File: pkgs/tools/networking/openresolv/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/tools/networking/openresolv/default.nix:31]:
```suggestion
      --prefix PATH "${lib.makeBinPath [ coreutils gnused dbus ]}"
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

**Comment by amnore** [comment]
Oops, it's been too long since this PR was first opened, and I don't use openresolv currently so I can't test the changes. Will close this for now. :(

**Pull request closed without merging**